config/rapidez/statamic.php

```diff
@@ -94,4 +94,9 @@
     'disabled_sites' => [
         // '{site_handle}'
     ],
+
+    'sitemap' => [
+        'prefix' => 'statamic_sitemap_',
```

Comment:
Maybe add an env value for this? That way we dan easily change it on production if we run into conflicts.
```suggestion
        'prefix' => env('STATAMIC_SITEMAP_PREFIX, 'statamic_sitemap_'),
```